example/metro.config.js

```diff
@@ -1,5 +1,17 @@
+const blacklist = require('metro-config/src/defaults/blacklist');
```

Comment:
Ran into this: https://github.com/facebook/react-native/issues/24065#issuecomment-537489786

Reply:
let's add this as a comment here

Reply:
Done